npm/runtime/src/helpers/decorate.js

```diff
@@ -1,250 +1,25 @@
-var toArray = require("./toArray.js");
-var toPropertyKey = require("./toPropertyKey.js");
-function _decorate(e, r, t, i) {
-  var o = _getDecoratorsApi();
-  if (i) for (var n = 0; n < i.length; n++) o = i[n](o);
-  var s = r(function (e) {
-      o.initializeInstanceElements(e, a.elements);
-    }, t),
-    a = o.decorateClass(_coalesceClassElements(s.d.map(_createElementDescriptor)), e);
-  return o.initializeClassElements(s.F, a.elements), o.runClassFinishers(s.F, a.finishers);
-}
-function _getDecoratorsApi() {
-  _getDecoratorsApi = function _getDecoratorsApi() {
-    return e;
+// Copy from https://github.com/microsoft/TypeScript/blob/d85767abfd83880cea17cea70f9913e9c4496dcc/src/compiler/factory/emitHelpers.ts#L730-L742
+
+const __decorate =
+  (this && this.__decorate) ||
```

Comment:
I think we can lose the `(this && this.__decorate) ||` line for all of these. TS seems to allow helpers to be present on global object, but none of our other helpers work like that. So, for the ESM versions, just `export default function(...) { ... }` should do it.

Reply:
Yes, I've made a change for this.